README.md

```diff
@@ -222,6 +222,17 @@ TODO(awong): Figure out sauce labs integrations. Not all e2e tests should always
 run on all browsers. That's wasteful. How do we determine what should be run on
 multiple browsers as opposed to on PhantomJS in Jenkins?
 
+#### E2E Troubleshooting
+Try running your `selenium` server manually:
+```
+$ java -jar <YOUR_HOME>/Github/vets-website/node_modules/selenium-server/lib/runner/selenium-server-standalone-3.1.0.jar
```

Comment:
It's nit-picky, but maybe the path would be better written as `<path to vets-website>/node_modules/...` or something. I only say that because I don't store my copy in `~/Github/` is all.